packages/contentstack-command/package.json

```diff
@@ -17,9 +17,9 @@
     "format": "eslint src/**/*.ts --fix"
   },
   "dependencies": {
-    "@oclif/core": "^1.21.0",
-    "@contentstack/cli-utilities": "^1.1.0",
+    "@contentstack/cli-utilities": "^1.2.0",
     "@contentstack/management": "^1.6.1",
+    "@oclif/core": "^2.3.0",
```

Comment:
@aman19K we can remove this package if not used directly

Reply:
@antonyagustine removed @oclif/core dependency from package.json